nvi-commons/src/test/java/no/sikt/nva/nvi/common/service/PublicationLoaderServiceTest.java

```diff
@@ -1,17 +1,61 @@
 package no.sikt.nva.nvi.common.service;
 
+import static no.sikt.nva.nvi.common.examples.ExamplePublications.CONTRIBUTOR_NO_AFFILIATION;
```

Comment:
The changes here were due to my "fixing" the import of one test and then reformatting…there is no real change here.